src/components/Link/tests/Link.test.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { mount, shallow } from 'enzyme';
+import Link from '../Link';
+
+describe('onClick()', () => {
+  it('is called when the Link is clicked', () => {
+      const spy = jest.fn();
+      shallow(<Link onClick={spy}>Test</Link>).simulate('click');
+      expect(spy).toHaveBeenCalled();
+  });
+});
+
+describe('external', () => {
+  it('should verify the external is false', () => {
+      const linkWrapper = mount(<Link external={false}/>);
+      expect(linkWrapper.prop('external')).toBe(false);
+  });
+
+  it('should verify the external is true', () => {
+      const linkWrapper = mount(<Link external={true}/>);
+      expect(linkWrapper.prop('external')).toBe(true);
+  });
+
+  it('should verify the external type on the Link', () => {
+      const linkWrapper = mount(<Link external={true}/>).prop('external');
```

Comment:
You don't need to pass a true value to a prop. If you just declare the prop with no value it will evaluate to true by default.

Example: 
`<Link external={true} />` === `<Link external />`